README.md

```diff
@@ -1,138 +1,78 @@
-[![Release](https://github.com/canonical/starbase/actions/workflows/release-publish.yaml/badge.svg?branch=main&event=push)](https://github.com/canonical/starbase/actions/workflows/release-publish.yaml)
-[![Documentation](https://github.com/canonical/starbase/actions/workflows/docs.yaml/badge.svg?branch=main&event=push)](https://github.com/canonical/starbase/actions/workflows/docs.yaml)
-[![test](https://github.com/canonical/starbase/actions/workflows/tests.yaml/badge.svg?branch=main&event=push)](https://github.com/canonical/starbase/actions/workflows/tests.yaml)
+# Rockcraft
 
-# starbase
+[![Rockcraft][rockcraft-badge]][rockcraft-site]
+[![Documentation Status][rtd-badge]][rtd-stable]
 
-The base repository for Starcraft projects.
+**Rockcraft** is the command-line tool for the rock container format, which are
+OCI-compliant images based on Ubuntu. It handles all the repetitive and boilerplate
+steps of building a rock, directing your focus to what really matters – the rock's
+content. From independent software vendors to container users of any experience level,
+Rockcraft is for anyone who wants to build production-grade rocks.
 
-## Description
+## Basic usage
 
-This template code is the basis for all future starcraft projects, and acts as the
-testbed for any major tooling changes that we want to make before propagating them
-across all projects.
+A rock's build configuration is stored in simple language as a project file called
+`rockcraft.yaml`.
 
-## Structure
+From the root of your container's codebase, Rockcraft creates a minimal `rockcraft.yaml` with:
 
-TODO
+```bash
+rockcraft init
+```
 
-## Migrate existing projects
+After you add all container's packages and dependencies to ``rockcraft.yaml``, bundle
+the rock with:
 
-1. Update this guide as you go along, if something is unclear or missing.
-2. Use ruff.
+```bash
+rockcraft pack
+```
 
-    1. Pull in the bare minimum `pyproject.toml` needed to use ruff.
+## Installation
 
-    2. Make your codebase pass with ruff. Commit after each step:
+Rockcraft is available on all major Linux distributions, Windows, and macOS.
```

Comment:
Today we don't test either Windows or macOS, so I don't know how confident we can state that we support those.

Reply:
b0733f8